### PR TITLE
feat(Org Settings): Contrast, Chrome, and Crumb improvements

### PR DESCRIPTION
**Pull request by ckj**
This is the first round of cleanup. It:

- Makes text colors and borders 3-4% darker.
- Converts Your teams, All Teams, and other Box-based wrappers to use our Panel component
- Introduces a `PanelItem` component for Flex-based panel rows that need padding and a bottom-border
- Removes the back button in favor of a top level sentry logo breadcrumb
- Moves beta notice below the content
- Locks textarea resize to vertical only

screen shot 2017-12-12 at 4 18 23 pm

### CODE REVIEW

**Comment by ckj** [comment]
@billyvg Do you mind to wire up the Logo to behave like the Back button?

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/utils/theme.js [hunk 1]
```diff
@@ -72,6 +68,18 @@ const theme = {
       textDark: '#5d3e3b',
     },
   },
+
+  type: {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/theme.js]: is there a better name for this, `type` is generically used in a lot of places

**Review by billyvg — APPROVED** [review]
can you squash before merging?

File: src/sentry/static/sentry/app/utils/theme.js (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/app/utils/theme.js]: Oof, good call. I'll think of something better.

**Pull request merged**